UserApi/UserApi/Services/UserAccountService.cs

```diff
@@ -60,32 +60,24 @@ public async Task AddUserToGroup(User user, Group group)
             var stringContent = new StringContent(JsonConvert.SerializeObject(body));
             var accessUri = $"{_graphApiSettings.GraphApiBaseUri}v1.0/{_graphApiSettings.TenantId}/groups/{group.Id}/members/$ref";
             var responseMessage = await _secureHttpRequest.PostAsync(_graphApiSettings.AccessToken, stringContent, accessUri);
-            if (responseMessage.IsSuccessStatusCode) return;
+            if (responseMessage.IsSuccessStatusCode)
+            {
+                return;
+            }
 
             var reason = await responseMessage.Content.ReadAsStringAsync();
             
             // if we failed because the user is already in the group, consider it done anyway 
-            if (reason.Contains("already exist")) return;
+            if (reason.Contains("already exist"))
```

Comment:
Is there not a status code that the api return that can be checked ? Like 409 Confilct... but probably not

Reply:
You would've thought so right :) I was equally surprised when I used this but it actually returns a BadRequest including this message.